perfkitbenchmarker/beam_benchmark_helper.py

```diff
@@ -137,7 +171,71 @@ def BuildMavenCommand(benchmark_spec, classname, job_arguments):
   cmd.append("-DintegrationTestPipelineOptions="
              "[{}]".format(','.join(beam_args)))
 
+  return cmd
+
+
+def _BuildPythonCommand(benchmark_spec, job_arguments):
```

Comment:
This method should take a class (or module) name as well, following the pattern set by the java method.